cli/dcos-spark/main.go

```diff
@@ -329,9 +337,11 @@ func handleCommands(app *kingpin.Application) {
 	status.Arg("submission-id", "The ID of the Spark job").Required().StringVar(&cmd.submissionId)
 
 	log := app.Command("log", "Retrieves a log file from a submitted Spark job").Action(cmd.runLog)
+	log.Flag("all", "Print log of completed or running Spark job").BoolVar(&cmd.logAll)
+	log.Flag("completed", "Print log of completed Spark job").BoolVar(&cmd.logCompleted)
 	log.Flag("follow", "Dynamically update the log").BoolVar(&cmd.logFollow)
-	log.Flag("lines_count", "Print the last N lines").
-		Default("10").UintVar(&cmd.logLines) //TODO "lines"?
+	log.Flag("lines", "Print the last N lines").
```

Comment:
I'd like to understand what the driver behind this change is, is it just the `TODO` line?

If so we should have `lines` and `lines_count` for this release, emit a deprecated warning in the CLI and flag this in the Release Notes.

This sort of thing can break scripts that users might have written using this sub-command.

Reply:
Yeah, changed it because of the `TODO` line. Agrees, I will revert this change.